### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Leon99** [SpamapS/terraform-provider-uptimerobot#4]
It doesn't work for some reason. Returns the following on creation attempt:
```
* uptimerobot_monitor.main: Server reports request not ok!
```
Is there a way to understand what went wrong?

**Comment by nlamirault** [comment]
I've got the same error.

**Comment by nlamirault** [comment]
It works with this provider : https://github.com/louy/terraform-provider-uptimerobot

**Comment by Leon99** [comment]
Thanks @nlamirault, I'll check it.

**Comment by SpamapS** [comment]
Yes actually I'm going to deprecate this one and suggest people use https://github.com/louy/terraform-provider-uptimerobot thanks for the nudge. ;)
